### DISCUSSION TITLE
DEFORUM EXTENSION

### DISCUSSION THREAD

**Discussion by humanplugged** [lllyasviel/stable-diffusion-webui-forge#299] [Ideas]
Would be possible having A1111 Deforum's extension with Forge?

On my M1 Max, it can render fast by using LCM or Turbo models with a lower resolution of 512x288 or a bit higher. 

I would love to know how much faster it could get using Forge. ♥️

Also, a tutorial on how to install Forge on a Mac would be very helpful. I'm having troubles to install it.

**Comment by catboxanon** [comment]
It's being looked into. https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/96#issuecomment-1950292908

**Reply by humanplugged** [reply]
Thank you catboxanon! =D
